tests/test_utils.py

```diff
@@ -142,6 +144,7 @@ def test_logging_ssl(caplog):
     ]
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Path separator problem")
```

Comment:
What's the issue here?

Reply:
tests will assert file path with `/`

Reply:
Okay, so let's fix up the test case so that we don't do that.

Presumably that's this line?...

```
assert ca_bundle is not None and ca_bundle.endswith("tests/test_utils.py")
```

Can we fix up those cases so we're not using literal `/` seperators.